src/views/migrationpolicies/list/components/MigrationPoliciesEmptyState/MigrationPoliciesEmptyState.tsx

```diff
@@ -18,7 +18,8 @@ import {
 import { ExternalLinkAltIcon } from '@patternfly/react-icons';
 
 import { migrationPoliciesPageBaseURL } from '../../utils/constants';
-
+const MigrationPolicyDocURL =
```

Comment:
If u already not extracting to costs file why not simply inline use at href?

Reply:
moved inline, thanks :)